test-utils/transaction-factory/src/simple_modes.rs

```diff
@@ -36,28 +36,29 @@
 use std::sync::Arc;
 
 use log::info;
+use client::Client;
 use client::block_builder::api::BlockBuilder;
 use client::runtime_api::ConstructRuntimeApi;
+use primitives::{Blake2Hasher, Hasher};
 use sr_primitives::traits::{Block as BlockT, ProvideRuntimeApi, One};
 use sr_primitives::generic::BlockId;
-use substrate_service::{
-	FactoryBlock, FullClient, ServiceFactory, ComponentClient, FullComponents
-};
 
 use crate::{Mode, RuntimeAdapter, create_block};
 
-pub fn next<F, RA>(
+pub fn next<RA, Backend, Exec, Block, RtApi>(
```

Comment:
Unfortunately no. The integration tests use a different runtime API struct than the one in the client. I don't really know whether this can be fixed, and if it can be fixed I think it should be by a different PR anyway.

Reply:
Okay :+1: